symphony/content/content.blueprintsdatasources.php

```diff
@@ -336,13 +337,25 @@ public function __form()
         $fieldset->appendChild($p);
         $group = new XMLElement('div');
 
-        $label = Widget::Checkbox('fields[redirect_on_required]', $fields['redirect_on_required'], __('The required parameter is missing'));
+        $label = Widget::Checkbox(
+            'fields[redirect_on_required]',
+            isset($fields['redirect_on_required']) ? $fields['redirect_on_required'] : null,
```

Comment:
`isset` is wrong here. See: http://php.net/manual/en/function.array-key-exists.php#example-6170

Reply:
Well, that's ok ;) It will do the same thing.

Reply:
@nitriques 
> It will do the same thing.

Only in this one case, but in others it is wrong. (See in the classes `Administration`,  `DateTimeObj`, `FieldManager`, `Symphony`, `Section`, etc.)
Always use `array_key_exists`, which makes your life easier – that should be the goal. 😃

Reply:
@froschdesign 
Well, we almost always treat non-set and set to null as being the same thing, so `isset()` does the job! But yeah, we might be wrong at a couple of places ;)